consoles/VNC.pm

```diff
@@ -378,7 +378,7 @@ sub _server_initialization ($self) {
     $self->_pixinfo(\%pixinfo);
     $self->_bpp($supported_depths{$self->depth}->{bpp});
     $self->_true_colour($supported_depths{$self->depth}->{true_colour});
-    $self->_do_endian_conversion($self->no_endian_conversion ? 0 : $server_is_big_endian != $client_is_big_endian);
+    $self->_do_endian_conversion(($server_is_big_endian && $client_is_big_endian) ? 1 : 0);
```

Comment:
And **if** this doesn't break our use case at SUSE, this is simpler written as:
```suggestion
    $self->_do_endian_conversion($server_is_big_endian && $client_is_big_endian);
```

Reply:
Updated. Thanks a lot!